cli/popper/commands/cmd_run.py

```diff
@@ -111,6 +111,11 @@
     hidden=True,
     default=popper.scm.get_git_root_folder()
 )
+@click.option(
+    '--engine-conf',
+    help='Specify the engine configuration file for a container.',
```

Comment:
maybe `File containing configuration options for the underlying container engine.`